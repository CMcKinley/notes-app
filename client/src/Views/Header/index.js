import React from "react";
import { makeStyles } from "@material-ui/core/styles";
import AppBar from "@material-ui/core/AppBar";
import Typography from "@material-ui/core/Typography";

const useStyles = makeStyles(() => ({
  header: {
    margin: "8px 16px"
  }
}));

const Header = () => {
  const classes = useStyles();

  // Render Header
  return (
    <AppBar>
      <Typography variant="h6" className={classes.header}>
        Notes
      </Typography>
    </AppBar>
  );
};

export default Header;
