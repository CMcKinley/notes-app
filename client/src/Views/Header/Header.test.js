import React from "react";
import renderer from "react-test-renderer";
import Header from "./index.js";

describe("Header", () => {
  it("Renders correctly", () => {
    const component = renderer.create(<Header />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
