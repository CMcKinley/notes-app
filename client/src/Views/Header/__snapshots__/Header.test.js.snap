// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Renders correctly 1`] = `
<header
  className="MuiPaper-root MuiPaper-elevation4 MuiAppBar-root MuiAppBar-positionFixed MuiAppBar-colorPrimary mui-fixed"
>
  <h6
    className="MuiTypography-root makeStyles-header-1 MuiTypography-h6"
  >
    Notes
  </h6>
</header>
`;
